conf/switches.conf.example

```diff
@@ -21,7 +22,7 @@ isolationRole = isolation
 macDetectionRole = macDetection
 voiceRole = voice
 inlineRole = inline
-VoIPEnabled = no
+VoIPEnabled = N
```

Comment:
Would switches.conf.defaults merge individual attributes in two sections with the same id in the config ?

If it doesn't then it would be next to useless because the default switch is always modified